src/main/kotlin/com/example/minibank/services/impl/FTServiceImpl.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.minibank.services.impl
+
+import com.example.minibank.config.FTProperties
+import com.example.minibank.config.FeatureToggle
+import com.example.minibank.services.FTService
+
+import org.springframework.stereotype.Service
+
+@Service
+class FTServiceImpl(val ftProps: FTProperties) : FTService {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line